packages/node_modules/@ciscospark/spark-core/src/index.js

```diff
@@ -42,6 +42,7 @@ export {default as RequestLoggerInterceptor} from './interceptors/request-logger
 export {default as RequestTimingInterceptor} from './interceptors/request-timing';
 export {default as SparkTrackingIdInterceptor} from './interceptors/spark-tracking-id';
 export {default as SparkUserAgentInterceptor} from './interceptors/spark-user-agent';
+export {default as RateLimitInterceptor} from './interceptors/rate-limit';
```

Comment:
I'll take a look, thanks.

Reply:
Done.

Reply:
I still don't see a change that adds the rate limit interceptor to the list of active interceptors.

Reply:
Right I hadn't pushed the commit since there was still disagreement on the use of ternary operators. I have added it to spark-js-sdk/packages/node_modules/@ciscospark/spark-core/src/spark-core.js, so should show up when I push.